modules/etcd-tuning-parameters.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="etcd-tuning-parameters_{context}"]
+= Setting tuning parameters for etcd
+
+You can set the control plane hardware speed to `""` (the default), `"Standard"`, or `"Slower"`.
+
+The default setting allows the system to decide which speed to use. This value enables upgrades from versions where this feature does not exist, as the system can select values from previous versions.
+
+By selecting one of the other values, you are overriding the default. If you see many leader elections due to timeouts or missed heartbeats and your system is set to `""` or `"Standard"`, set the hardware speed to `"Slower"` to make the system more tolerant to the increased latency.
+
+:FeatureName: Tuning etcd latency tolerances
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-changing-hardware-speed-tolerance_{context}"]
+== Changing hardware speed tolerance
+
+To change the hardware speed tolerance for etcd, complete the following steps.
+
+.Procedure
+
+. Check to see what the current value is by entering the following command:
++
+[source, terminal]
+----
+$ oc describe etcd/cluster | grep "Control Plane Hardware Speed"
+----
++
+.Example output
+[source,terminal]
+----
+Control Plane Hardware Speed:  <VALUE>
+----
++
+[NOTE]
+====
+If the output is empty, the field has not been set and should be considered as the default ("").
+====
+
+. Change the value by entering the following command. Replace `<value>` with one of the valid values: `""`, `"Standard"`, or `"Slower"`:
++
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"controlPlaneHardwareSpeed": "<value>"}}'
+----
++
+The following table indicates the heartbeat interval and leader election timeout for each profile. These values are subject to change.
++
+|===
+| Profile | ETCD_HEARTBEAT_INTERVAL | ETCD_LEADER_ELECTION_TIMEOUT
+| `""` | Varies depending on platform | Varies depending on platform
+| `Standard` | 100 | 1000
+| `Slower` | 500 | 2500
+|===
+
+. Review the output:
++
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
+----
++
+If you enter any value besides the valid values, error output is displayed. For example, if you entered `"example"` as the value, the output is as follows:
++
+.Example output
+[source,terminal]
+----
+The Etcd "cluster" is invalid: spec.controlPlaneHardwareCSpeed: Unsupported value: "example": supported values: "", "Standard", "Slower"
```

Comment:
For the sake of clarity, consider using something instead of"example" for the first value in the first sentence. This way, the reader won't have to recognize the diffeent uses of "example" as a value and as an output type.

Reply:
Thanks for your review and feedback, @nalhadef! I chatted with engineering about which value to use instead of "example", and I updated it to "Faster".